.clang-format

```
@@ -0,0 +1,14 @@
+BasedOnStyle: "Google"
+
+AlignAfterOpenBracket:        Align
+AlignArrayOfStructures:       Left
+AlignConsecutiveAssignments:  AcrossComments
+AlignConsecutiveDeclarations: Consecutive
+AlignEscapedNewlines:         true
+AlignOperands:                Align
```

We did this in PETSc so multiline expressions become
```c
  f[i] = long_expression  // descriptions
       + with * many      // of the terms
       - terms;
```
which is how I'd format it in LaTeX, versus
```c
  f[i] = long_expression + // descriptions
         with * many -     // of the terms
         terms;
```
where I think the `+` and `-` signs are less noticeable. The downside is imperfect column alignment inside parentheses (relevant if you have multiline expressions where columns are similar).

```suggestion
AlignOperands:                AlignAfterOperator
```